dbms/src/Storages/KVStore/MultiRaft/ApplySnapshot.cpp

```diff
@@ -95,23 +95,54 @@ void KVStore::checkAndApplyPreHandledSnapshot(const RegionPtrWrap & new_region,
             if (overlapped_region.first != region_id)
             {
                 auto state = getProxyHelper()->getRegionLocalState(overlapped_region.first);
-                if (state.state() != raft_serverpb::PeerState::Tombstone)
+                auto extra_msg = fmt::format(
+                    "state={}, tiflash_state={}, new_region_state={}",
```

Comment:
```suggestion
                    "state={} tiflash_state={} new_region_state={}",
```

Reply:
Given the following log format as an example, I prefer that we do not remove the comma. Since the space is not enough to make a delimiter
```
[2024/08/19 11:44:40.156 +08:00] [INFO] [ApplySnapshot.cpp:135] ["range of region_id=2 is overlapped with `Applying` region_id=1, state=state: Applying region { id: 1 start_key: \"t\\200\\000\\000\\000\\000\\000\\000\\377\\037_r\\200\\000\\000\\000\\000\\377\\000\\000\\000\\000\\000\\000\\000\\000\\372\" end_key: \"t\\200\\000\\000\\000\\000\\000\\000\\377\\037_r\\200\\000\\000\\000\\000\\377\\000\\000\\001\\000\\000\\000\\000\\000\\372\" }, tiflash_state=region { id: 1 start_key: \"t\\200\\000\\000\\000\\000\\000\\000\\377\\037_r\\200\\000\\000\\000\\000\\377\\000\\000\\000\\000\\000\\000\\000\\000\\372\" end_key: \"t\\200\\000\\000\\000\\000\\000\\000\\377 _r\\200\\000\\000\\000\\000\\377\\000\\000\\000\\000\\000\\000\\000\\000\\372\" region_epoch { version: 6 } peers { id: 1 } peers { id: 2 } }, new_region_state=region { id: 2 start_key: \"t\\200\\000\\000\\000\\000\\000\\000\\377\\037_r\\200\\000\\000\\000\\000\\377\\000\\000\\001\\000\\000\\000\\000\\000\\372\" end_key: \"t\\200\\000\\000\\000\\000\\000\\000\\377\\037_r\\200\\000\\000\\000\\000\\377\\000\\000\\004\\000\\000\\000\\000\\000\\372\" peers { id: 2 } }"] [thread_id=1]
```